src/lang/en-US/AdminPage.ts

```diff
@@ -121,6 +121,17 @@ export const AdminPage: Record<string, string> = {
     'admin.alerts.table.noContent.message': `To begin receiving email notifications for a prefix you admin, click "Configure Notifications" above to create a subscription.`,
     'admin.alerts.table.label.alertMethod': `Notification Method`,
 
+    'admin.dataPlanes.private.header': `Private Data Planes`,
+    'admin.dataPlanes.table.aria.label': `Data Planes Table`,
+    'admin.dataPlanes.table.columns.scope': `Scope`,
+    'admin.dataPlanes.table.columns.name': `Name`,
```

Comment:
Sharing that _Name_ does exist in the common fields section of _CommonMessages.ts_ (i.e., `'entityName.label'`).